ykrt/src/compile/jitc_yk/codegen/x64/lsregalloc.rs

```diff
@@ -227,6 +227,10 @@ impl<'a> LSRegAlloc<'a> {
         self.stack.size()
     }
 
+    pub(crate) fn stack_size(&mut self) -> usize {
```

Comment:
Add a doc string (even I can't remember if this is bytes or #entries, and I wrote the code I think!).

Reply:
[0022632](https://github.com/ykjit/yk/pull/1422/commits/00226328b8a1cfff73af85e11612135268003639)